code/game/objects/machinery/computer/general_air_control.dm

```diff
@@ -91,6 +91,31 @@
 	var/pressure_setting = ONE_ATMOSPHERE * 45
 	circuit = /obj/item/circuitboard/computer/air_management/tank_control
 
+/obj/machinery/computer/general_air_control/large_tank_control/lowerdeck
+	frequency = 1443
+	input_tag = "waste_lower_in"
+	name = "Lower Deck Waste Tank Control"
+	output_tag = "waste_lower_out"
+	sensors = list("waste_sensor" = "Tank")
```

Comment:
Due to atms being nonfunctional, delete the frequencies (they should be defines anyway), the input and output tags, and sensors